woocommerce/woocommerce
# [ci-jobs] Adds --list and --json options to save jobs output

### Changes proposed in this Pull Request:

Adds two new options to the `ci-jobs` cli tool, to help better understand what jobs run in which conditions.

- `--list` will print the jobs output in console in a table for a more readable form
- `--json` will save the jobs output in a file called `jobs.json`

The json file will also contain the `baseRef` and the `event` information.


### How to test the changes in this Pull Request:

Run the ci-jobs tool using both `--list` and `--json` options:

Should list all jobs:

```
pnpm utils ci-jobs --list
```

Should save all jobs in `jobs.json`:
```
pnpm utils ci-jobs --json
```

Should list and save all jobs that should run for this PR:
```
pnpm utils ci-jobs -r origin/trunk -e pull_request --list --json
```

Should list all jobs that should run as release-checks:
```
pnpm utils ci-jobs -e release-checks --list --json
```
